fcrepo-mysql.yml

```diff
@@ -0,0 +1,36 @@
+version: "3.6"
+
+networks:
+  appnetwork:
+    driver: bridge
+
+services:
+  fcrepo:
+    container_name: fcrepo
+    build: 
+      context: ./docker/services/fcrepo
+      args:
+        FEDORA_VERSION: 4.7.5
+        FEDORA_TAG: 4.7.5
+        ModeshapeConfig: jdbc-mysql
+        JDBCConfig: -Dfcrepo.mysql.username=fcrepoadmin -Dfcrepo.mysql.password=fcrepoadminpw -Dfcrepo.mysql.host=mysql.docker.local -Dfcrepo.mysql.port=3306 
```

Comment:
Shouldn't these username/password/host/etc. settings reference the environment variables, to ensure they match up?

Also, is the database name (`MYSQL_DATABASE`/`fcrepo`) hard-coded or otherwise assumed?  Nothing seems to reference it in the `JDBCConfig` setting

Reply:
For now, it is just a demonstration for JDBC object store https://wiki.duraspace.org/display/FEDORA5x/Configuring+JDBC+Object+Store. The database name fcrepo is a sample database.